force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +309,11 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
+       for(String str : inputList){
+            stringSet.add(str);
+        }
 
-        return null; // Replace null with the variable you used to store the result
+        return stringSet; 
```

Comment:
Good but you could do this in one line with the Set constructor. 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_set.htm